### DISCUSSION TITLE
automatische Zeiteinstellung über den Zeitserver

### DISCUSSION THREAD

**Discussion by sebther** [tbnobody/OpenDTU#539] [Q&A] [answered]
Zuerst einmal ein "Hallo an Alle" und einen riesigen Daumen hoch für dieses Projekt!!!
Seit letztem Wochende bin ich nun auch stolzer Besitzer solch einer Open DTU HW/SW und es läuft echt super :-)
Nun bastle ich schon daran, dass ich die Daten auch über MQTT weiterverarbeiten kann.

Aber vorher zu meiner Frage bzw. zu meiner Bitte um Hilfe:
Mir ist aufgefallen, dass mein OpenDTU sich nicht mit dem Zeitserver synchronisiert, wenn ich es "neu" am Computer anstecke.
Ich muss dann immer wieder die Zeitsynchronisierung händisch durchführen - dann läuft es.
Meiner Meinung nach habe ich alles richtig konfiguriert - siehe dazu die Bilder:
![grafik](https://user-images.githubusercontent.com/123261571/214533131-e96a01b9-e674-43c7-b702-c1a816ee3502.png)
Auch sollte das Netzwerk richtig eingestellt sein...
![grafik](https://user-images.githubusercontent.com/123261571/214533436-06cced25-f6aa-437d-b991-f4f779a1a61d.png)

Kennt jemand dieses Problem und was kann ich dagegen tun?

Danke schon vorab für euren Rückmeldungen!

LG Helmut

**Comment by battifreund** [comment]
Moin! Du hast als DNS-Server 0.0.0.0 eingetragen. Manche Router routen diese Adresse zum Default-DNS-Server, den Du vom Provider bekommst, wenn ich mich richtig erinnere.

Meine Hypothese:
Der Router hat die Route zum Zeitpunkt der ersten Abfrage noch nicht korrekt zugewiesen. Deshalb schlägt die erste Anfrage fehl, Dein späterer manueller Abruf geht aber durch.

Trag doch mal 8.8.8.8 als DNS-Server ein.

**Comment by Juergen2453** [comment, marked as answer]
Du hast keinen DNS-Server im Netzwerk (Statische IP Konfiguration > DNS Server) eingetragen.
oder du trägst eine IP z.B. (Uni Düsseldorf) 134.99.128.80 im NTP Konfiguration > Zeitserver ein.

@battifreund war schneller

**Comment by sebther** [comment]
Hallo an battifreund und Juergen2453!

Perfekt - nun läuft es 
Im Nachgang ist es mir jetzt auch klar..... 

VIEEEEEELEN DANK!!!

LG Helmut
